src/App.vue

```diff
@@ -100,10 +98,12 @@ img {
 <style>
 p {
   max-width: 75ch; /* better readability supposedly */
+  margin-bottom: 16px;
```

Comment:
this seems to have been globally applied somehow before

Reply:
<img width="433" alt="image" src="https://github.com/user-attachments/assets/80d121a3-341d-47dd-ab6f-a07c22945ee2" />